app/registries/menu/item.rb

```diff
@@ -18,7 +18,7 @@ def initialize(name, options)
       @child_menus = options[:children]
       @last = options[:last] || false
       @context =  options[:engine] || Rails.application
-      @turbolinks = options.fetch(:turbolinks, true)
+      @turbolinks = false
```

Comment:
do we even want to keep this option? doesn't seem like we should given it's not used. iirc this was originally introduced to allow skipping turbolinks on pages we know will cause a full page reload when loaded (due to js etc) but i'm not sure if it has ever been actually used.

Reply:
dropped the option. You're right, we do not want to keep it.